### ISSUE TITLE
Python: Silent crash when calling Editor.Getbreakpoints outwith main thread

### ISSUE THREAD

**Issue by onelivesleft** [slynch8/10x#1685]
```python
import threading
from N10X import Editor

def ByeBye():
    def foo():
        breakpoints = Editor.GetBreakpoints()
    thread = threading.Thread(target=foo)
    thread.start()
```

Call command ByeBye to kill 10x

**Comment by onelivesleft** [comment]
Just to report; you said most of the calls don't work except on the main thread, and with that in mind it's easy enough to move all the interaction logic there.  No point making this specific call work on other threads unless you're going to do it for everything.  It would be good if every call had a simple guard at the top that threw a CanOnlyCallOnMainThread exception or something, so it was clear what was going wrong; when I hit this I thought it was something I was doing with my spawned process that was crashing 10x, and only when I tracked it down did I find it in this unexpected place.

**Comment by slynch8** [comment]
ok, that's good. I'll make it throw as you say.

**Comment by slynch8** [comment]
fixed (1.0.35)
